source/_patterns/02-organismer/50-ekspanderbart-panel/61-andre-med-rettigheter-liste-visning.json

```diff
@@ -0,0 +1,461 @@
+    "button-class": "a-btn a-btn-action a-fullWidthBtn",
+    "action-button-text": "Legg til ny person eller virksomhet",
+    "onclick": "location.href='link.sider-portal-andremedrettigheter-legg-til-ansatt'",
+    "icon-prefix": "ai",
+    "icon-class": "ai-plus",
+    "icon-extra-class": ""
+  },
+  "sok-etter-person": {
+    "search-target": "employeeTableId",
+    "search-algorithm": "show-and-highlight",
+    "display": true,
+    "text-input-id": "randomIdTwo",
+    "top-label": "Finn skjema eller tjeneste",
+    "isHidden": true,
+    "text-input-placeholder": "Søk på person eller virksomhet",
+    "input-type": "search",
+    "isRequired": false,
+    "isLarge": false,
+    "isInput-group": "input-group",
+    "form-input-hasButton": {
+      "button-icon": {
+        "icon-prefix": "ai",
+        "icon-class": "ai-search"
+      }
+    },
+    "form-input-Icon": false
+  },
+  "liste-unummerert-expandable":{
+    "is-large":true,
+    "rows":[
+      { 
+        "additional-classes":"a-list-parentRightHolder",
+        "onclick":false,
+        "expandedLink":{
+          "data-toggle":"",
+          "target-Link":"#"
+        },
+        "columns":[ 
+            { 
+              "additional-classes":"col-2 col-sm-1 col-lg-2",
+              "onclick":false,
+              "column-items":[ 
+                  { 
+                    "column-icon":{ 
+                        "icon-prefix":"ai",
+                        "icon-class":"ai-private",
+                        "icon-extra-class":""
+                    }
+                  }
+              ]
+            },
+            { 
+              "additional-classes":"col-6 col-sm-4 col-lg-4",
+              "onclick":false,
+              "column-items":[ 
+                  { 
+                    "column-text":{ 
+                        "classes":"a-js-sortValue",
+                        "text":"Person x"
+                    }
+                  }
+              ]
+            },
+            { 
+              "additional-classes":"col-md-3 d-none d-md-block col-lg-3",
+              "onclick":false,
+              "column-items":[
+                { 
+                  "column-text":{ 
+                      "classes":"a-js-sortValue",
+                      "text":"Regnskapsfører"
+                  }
+                }
+              ]
+            },
+            { 
+              "additional-classes":"col-4 col-sm-7 a-hiddenWhenSelected col-lg-3",
+              "onclick":false,              
+              "column-items":[ 
+                {
+                  "column-icon":{ 
+                    "icon-prefix":"reg",
+                    "icon-class":"reg-circle-exclamation",
+                    "icon-extra-class":"notificationIcon"
+                  }
+                }, 
+                { 
+                  "column-text":{ 
+                    "classes":"a-js-sortValue deleteOrdelegateText",
+                    "text":"Ber om Tilgang"
```

Comment:
har man valgt å endre denne teksten?
i dag ser det slik ut:
![image](https://user-images.githubusercontent.com/5472011/66898786-1eda5b80-effa-11e9-8670-8ec039868a08.png)

mens det nå er designimplementert slik:
![image](https://user-images.githubusercontent.com/5472011/66898839-3addfd00-effa-11e9-960d-0b526e436a12.png)

jeg mener både test, farge og størrelse bør uansett matche slik at det ser likt ut som for de to radene under

Reply:
ser av figma at dette ikke er tenkt på, men kanskje Maria kan svare på det? om man har tatt et valg eller bare ikke tenkt på det?

Reply:
fixed